cmd/agent/api/internal/agent/agent.go

```diff
@@ -388,7 +388,7 @@ func workloadList(w http.ResponseWriter, verbose bool) {
 	response := workloadmeta.GetGlobalStore().Dump(verbose)
 	jsonDump, err := json.Marshal(response)
 	if err != nil {
-		setJSONError(w, log.Errorf("Unable to marshal workload list response: %w", err), 500)
+		setJSONError(w, log.Errorf("Unable to marshal workload list response: %v", err), 500)
```

Comment:
👍 
It would be nice if `log.Errorf` supported [`%w`](https://pkg.go.dev/fmt#Errorf), but looks like it doesn't today due to log sanitization [here](https://github.com/DataDog/datadog-agent/blob/95e1e892fdaa21911bed770d63332ef57baf0d06/pkg/util/log/log.go#L407-L410).